apps/laboratory/src/pages/library/solana.tsx

```diff
@@ -25,7 +23,7 @@ const modal = createWeb3Modal({
   termsConditionsUrl: 'https://walletconnect.com/terms',
   privacyPolicyUrl: 'https://walletconnect.com/privacy',
   customWallets: ConstantsUtil.CustomWallets,
-  wallets: [new BackpackWalletAdapter(), new HuobiWalletAdapter(), new SolflareWalletAdapter()]
+  wallets: []
```

Comment:
Why these are discarted?

Reply:
I think this is redundant, wallet standard add them by itself